src/containers/IntentService.jsx

```diff
@@ -37,6 +37,9 @@ export default class IntentService extends Component {
         if (!konnector) {
           throw new Error(`Unknown konnector`)
         }
+        if (konnector.oauth === true) {
```

Comment:
```js
if (konnector.oauth)
```
is enough IMHO.

By the way, if we provide a default size for OAuth konnectors, we should also provide one for Non-OAuth konnectors.